discordrb.gemspec

```
@@ -31,17 +31,17 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'discordrb-webhooks', '~> 3.4.2'
 
-  spec.required_ruby_version = '>= 2.7'
+  spec.required_ruby_version = '>= 3.0'
 
   spec.add_development_dependency 'bundler', '>= 1.10', '< 3'
   spec.add_development_dependency 'rake', '~> 13.0'
-  spec.add_development_dependency 'redcarpet', '~> 3.5.0' # YARD markdown formatting
-  spec.add_development_dependency 'rspec', '~> 3.11.0'
-  spec.add_development_dependency 'rspec_junit_formatter', '~> 0.5.1'
+  spec.add_development_dependency 'redcarpet', '~> 3.6.0' # YARD markdown formatting
+  spec.add_development_dependency 'rspec', '~> 3.12.0'
+  spec.add_development_dependency 'rspec_junit_formatter', '~> 0.6.0'
   spec.add_development_dependency 'rspec-prof', '~> 0.0.7'
-  spec.add_development_dependency 'rubocop', '~> 1.36.0'
+  spec.add_development_dependency 'rubocop', '~> 1.48.0'
```

```suggestion
  spec.add_development_dependency 'rubocop', '~> 1.52.0'
```

Just need to update to the latest RuboCop version and apply any new changes it looks like.

---

Though it looks like there's some false positives due to rubocop/rubocop#11915, so watch out for these.

---

We'd been on 1.36 for over 7 months, and in Rubocop's changelogs there are constant "false positives" and even "false negatives" between each version.

I just put the most recent one at a given time (when I started working on this PR), which solved the problems in Ruby 3.x

Not to mention that before this PR becomes part of "main", there'll be a certain amount of time in between. I'm not going to come back and increment the version number regularly, I'll do it if it's really necessary, but in due time.